filebeat/templates/default/filebeat.yml.erb

```diff
@@ -0,0 +1,16 @@
+filebeat.prospectors:
+- input_type: log
+
+  paths:
+  <% @paths.each do |path| %>
+    <%= "- #{path}" %>
+  <% end %>
+
+  json.message_key: event
+  json.keys_under_root: true
+
+output.elasticsearch:
+  hosts: <%= @hosts %>
```

Comment:
should this be nested as a loop (as above with `paths`)? unclear from just reading it if this stringifies to valid yaml with multiple values